usl_pipeline/usl_lib/tests/transformers/test_feature_raster_transformers.py

```diff
@@ -0,0 +1,186 @@
+import numpy
+from numpy import testing
+from shapely import geometry
+
+from usl_lib.shared import geo_data
+from usl_lib.transformers import feature_raster_transformers
+
+
+def bbox_polygon(x1: float, y1: float, x2: float, y2: float) -> geometry.Polygon:
+    return geometry.Polygon([(x1, y1), (x2, y1), (x2, y2), (x1, y2), (x1, y1)])
+
+
+def test_transform_to_feature_raster_layers():
+    infiltration_configuration = geo_data.DEFAULT_INFILTRATION_CONFIGURATION
+    max_hydraulic_conductivity = max(
+        [item.hydraulic_conductivity for item in infiltration_configuration.items]
+    )
+    max_wetting_front_suction_head = max(
+        [item.wetting_front_suction_head for item in infiltration_configuration.items]
+    )
+    max_effective_porosity = max(
+        [item.effective_porosity for item in infiltration_configuration.items]
+    )
+    max_effective_saturation = max(
+        [item.effective_saturation for item in infiltration_configuration.items]
+    )
+
+    header = geo_data.ElevationHeader(
+        col_count=3,
+        row_count=4,
+        x_ll_corner=0.0,
+        y_ll_corner=0.0,
+        cell_size=1.0,
+        nodata_value=-9999.0,
+    )
+    data = numpy.asarray(
```

Comment:
can just call `array` here rather than `asarray` -- `asarray` is meant for coercing values to arrays rather than constructing, although it does work here.

Reply:
Done